shell/platform/windows/flutter_window.cc

```diff
@@ -277,4 +281,21 @@ PointerLocation FlutterWindow::GetPrimaryPointerLocation() {
   return {(size_t)point.x, (size_t)point.y};
 }
 
+void FlutterWindow::OnThemeChange() {
```

Comment:
Please add unit tests for both these methods. You may need to push the "is high contrast enabled" logic down to `Window` for mocking purposes.

Reply:
Which "both these methods" are you referring to? And what would the unit test check for, exactly? The outcome of calling `OnThemeChange` depends on whether high contrast is enabled on the system running it.

Reply:
Both of the methods you added: `OnThemeChanged` and `SendInitialAccessibilityFeatures`.

You should check that `OnThemeChanged` and `SendInitialAccessibilityFeatures` call `UpdateHighContrastEnabled` if high contrast is enabled or disabled, and, they don't call `UpdateHighContrastEnabled` for systems that don't support `SPI_GETHIGHCONTRAST`

> The outcome of calling OnThemeChange depends on whether high contrast is enabled on the system running it.

Right, hence this bit: `You may need to push the "is high contrast enabled" logic down to Window for mocking purposes.`

Reply:
I do not understand what the connection is between `The outcome of calling OnThemeChange depends...` and `You may need to push the "is high contrast enabled" logic...`. I am unclear as to what we would want to check in the case of a unittest for the `OnThemeChanged` function to verify it passes. What would we use as an indicator of a successful or failed test?

Reply:
For example, see how scrolling speed works:

https://github.com/flutter/engine/blob/ebf65036d1bf0d7fefc55f6d1e21f61f85455752/shell/platform/windows/flutter_window.cc#L226-L237

This calls `GetScrollOffsetMultiplier`, which can be mocked for testing purposes:

https://github.com/flutter/engine/blob/2bfad7964202a862183ecca2664ac1a2caef95e6/shell/platform/windows/flutter_window_unittests.cc#L350-L368

Reply:
Please check if `FlutterWindowTest.OnThemeChange` accomplishes what we need this unit test to do?